src/playground/ec/.storybook/preview-head.html

```diff
@@ -47,6 +57,7 @@
 <script>
   // https://github.com/storybookjs/storybook/issues/6113#issuecomment-473965255
   function runOnPageChange() {
+    ECL.autoInit().destroy();
```

Comment:
Do we really need to destroy everything on each click on in a different story..? I don't necessarily see the need for this, but i might not consider something..

Reply:
Yes  first of all to lighten the global `ECL.components` array which grows with each component instantiation. And also because I noticed that by switching rtl and navigate between the stories, if the destroy is not done it creates problems..